src/app/app.component.ts

```diff
@@ -233,4 +240,10 @@ export class AppComponent extends LogoutBasePageComponent implements OnInit {
   openSideMenu = (): void => {
     this.store$.dispatch(SideMenuOpened());
   };
+
+  filterPagesOnRole = (pages: Pages[]): Pages[] => {
+    if (this.appConfigProvider.getAppConfig().role === ExaminerRole.DLG) {
+      return pages.filter((page) => page.title !== UNUPLOADED_TESTS_PAGE);
+    } return pages;
+  };
```

Comment:
This we should probably use a filter instead of this, we can also adjust the logic which will make excluding based on role a little easier, for example... 

If we update the `Pages` interface to be
```
export interface Pages {
  title: string;
  descriptor: string;
  showUnSubmittedCount?: boolean;
  hideWhenRole?: ExaminerRole[];
}
```

Then the `Pages` array would look something like this
```
  Pages: Pages[] = [
    { title: DASHBOARD_PAGE, descriptor: 'Dashboard', },
    {
      title: UNUPLOADED_TESTS_PAGE,
      descriptor: 'Unsubmitted Tests',
      showUnSubmittedCount: true,
      hideWhenRole: [ExaminerRole.DLG],
    },
  ];
```

The `html` would look like 
```
 <ion-item *ngFor="let page of Pages | roleFilter" class="menu-item" detail="true" (click)="navPage(page)">
```

Where `roleFilter` is a pipe which is the recommended way for the type of filtering we are doing here, this would work like
```
import { Pipe, PipeTransform } from '@angular/core';
import { Pages } from '@app/app.component';
import { isAnyOf } from '@shared/helpers/simplifiers';
import { AppConfigProvider } from '@providers/app-config/app-config';

@Pipe({
  name: 'roleFilter',
})
export class RoleFilterPipe implements PipeTransform {

  constructor(private appConfigProvider: AppConfigProvider) {
  }

  transform(pages: Pages[]): Pages[] {
    const role = this.appConfigProvider.getAppConfig()?.role;
    if (!role) {
      return pages;
    }

    return pages.filter(
      (page) => !isAnyOf(role, (page.hideWhenRole || [])),
    );
  }
```